src/magicbox/MagicBox.ts

```diff
@@ -105,6 +106,7 @@ export class MagicBoxInstance {
 
   public setText(text: string) {
     $$(this.element).toggleClass('magic-box-notEmpty', text.length > 0);
+    this.magicBoxClear.toggleTabindex(text.length > 0);
```

Comment:
Just to make sure, do all magixBox text changes go through this method? For example, aside from user input, is it used when selecting a value from the suggestions dropdown?

Reply:
It is not, just like the Clear isn't also showed on selecting a dropdown value, maybe there is an improvement here. But for now it seems consistent with the display of the element

Reply:
Okay, I would assume that is a bug, but we should not fix that here, rather in a separate jira/PR. I was thinking whether there is some event (or other mechanism) to know if there is text in the magic box or not? That would allow to shifting from a manual call to an event handler inside `MagicBoxClear.ts`. If not, then we can continue using the current approach.